roles/ohpc_add_easybuild/tasks/main.yaml

```diff
@@ -23,7 +25,7 @@
 # Ideal solution is to provide a python specific to EasyBuild
 # likely via software collections
 - name: Fix EasyBuild setuptools dependency on version > 17.1
-  shell: pip install -U setuptools
+  shell: pip3 install -U pip setuptools wheel
```

Comment:
Is it necessary to install pip when using pip3.  What is the end result?

Reply:
I think what we have right now is correct. This `pip3 install -U pip` upgrades pip3 version to latest